src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -131,24 +133,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return $this->display($input, $output, $io, $filesInfo);
     }
 
-    private function getFilesInfo(array $filenames): array
+    private function getFilesInfo(array $filenames, array $excludes): array
     {
         $filesInfo = [];
         foreach ($filenames as $filename) {
-            foreach ($this->findFiles($filename) as $file) {
+            foreach ($this->findFiles($filename, $excludes) as $file) {
                 $filesInfo[] = $this->validate(file_get_contents($file), $file);
             }
         }
 
         return $filesInfo;
     }
 
-    protected function findFiles(string $filename): iterable
+    protected function findFiles(string $filename, array $excludes): iterable
```

Comment:
Adding a new argument to a protected method is a BC break. It's already biting us here (see the failing test cases). I suggest a solution like this: https://github.com/symfony/symfony/pull/53745

Reply:
@xabbuh 👋🏻 copy that, I had in mind only public methods
i will check your link ASAP and try to provide a better approach